optimum/commands/export/tflite.py

```diff
@@ -116,6 +117,97 @@ def parse_args_tflite(parser):
         help=f"Audio tasks only. Audio sequence length {doc_input}",
     )
 
+    quantization_group = parser.add_argument_group("Quantization")
+    quantization_group.add_argument(
+        "--quantize",
+        choices=[e.value for e in QuantizationApproach],
+        type=str,
+        default=None,
+        help="The method of quantization to perform. No quantization will happen if left unspecified.",
```

Comment:
Maybe detail the choices here or have a link to a reference

Reply:
Done!